oak/server/rust/oak_runtime/src/metrics/server.rs

```diff
@@ -83,6 +83,8 @@ async fn make_server(port: u16) -> Result<(), hyper::error::Error> {
     server.await
 }
 
+// Start running a metrics server on the given port, running until the `notify_receiver` is
```

Comment:
Two suggestions (happy to discuss if necessary):

- make all comments "doc comments" (`///`), even on private symbols, just in case we make them public later; also they will come up when generating documentation for private symbols, which we can do locally (I do sometimes, at least), which will also allow `cargo doc` to check them
- start docs with a verb in the 3rd singular person ("Starts running..."). There does not seem to be a strong guideline, but the Rust by example book (https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html) and various high-profile crates use this style. No need to change everything at once in our repo, but perhaps we can follow this convention going forward, and write it down somewhere for reference.

Reply:
Making all top-level comments into doc comments makes sense; I've done a quick pass (but not gone through absolutely everywhere).

Wording-wise, we're about 50-50 between imperative ("Start running...") and declarative ("Starts running...") clause types<sup>1</sup>.  I tend towards the imperative myself, as does [Python](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), but [Java](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide) and [Go](https://blog.golang.org/godoc) officially recommend the declarative.  For Rust, [RFC 505](https://github.com/rust-lang/rfcs/blob/master/text/0505-api-comment-conventions.md) quietly uses imperative, but [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md) explicitly recommends declarative.

Having just been nerd-sniped, I personally think that it's not worth worrying about.

But if you want to make everywhere consistent and add a style guide, go ahead!

<hr/>
1: Chapter 9, "A Student's Introduction to English Grammar", Rodney Huddleston and Geoffrey K. Pullum